lib/pdk/util.rb

```diff
@@ -248,16 +248,18 @@ def targets_relative_to_pwd(targets)
     module_function :targets_relative_to_pwd
 
     # TO-DO: Refactor replacement of lib/pdk/module/build.rb:metadata to use this function instead
-    def module_metadata
+    # @param module_path [String] The path to the root of the module. Default is determine the module root automatically
+    def module_metadata(module_path = nil)
       require 'pdk/module/metadata'
-
-      PDK::Module::Metadata.from_file(File.join(module_root, 'metadata.json')).data
+      module_path = module_root if module_path.nil?
```

Comment:
```suggestion
      module_path ||= module_root
```

Or just use `module_path || module_root` in the `File.join` below

Reply:
Will change.

Reply:
Fixed